app/src/main/java/com/illiouchine/jm/ui/screen/PollResultScreen.kt

```diff
@@ -104,90 +105,92 @@ fun ResultScreen(
 
             val amountOfProposals = result.proposalResultsRanked.size
             result.proposalResultsRanked.forEachIndexed { displayIndex, proposalResult ->
-                Column(
-                    modifier = Modifier
-                        .clickable {
-                            // Clicking on the last is clicking on the penultimate.
-                            val clickedIndex = if (displayIndex == amountOfProposals - 1) {
-                                displayIndex - 1
-                            } else {
-                                displayIndex
-                            }
-                            // Behaves like an exclusive toggle
-                            if (isAnyProfileSelected && selectedProfile.value == clickedIndex) {
-                                isAnyProfileSelected = false
-                            } else {
-                                isAnyProfileSelected = true
-                                selectedProfile.value = clickedIndex
+                if (proposalResult.analysis.totalSize.compareTo(BigInteger.ZERO) > 0) {
```

Comment:
Because of the change in indentation, the diff is horrible, but only this line has changed.